content-security-policy/support/testharness-helper.sub.js

```diff
@@ -41,6 +41,13 @@ function generateUrlWithPolicies(host, policy) {
     return url + "?policy=" + encodeURIComponent(policy);
 }
 
+function generateUrlWithAllowCSPFrom(host, allowCspFrom) {
+  var url = generateURL(host, PolicyHeader.ALLOW_CSP_FROM);
+  if (!allowCspFrom)
+    return url + "?allow_csp_from=";
```

Comment:
I also added a check in the py file `request.GET["allow_csp_from"]`` but indeed then both versions do not send Allow-CSP-From header. 
I did not remove `"?allow_csp_from="` because otherwise GET parsing is weird since "&id" is added to url in `assert_iframe_with_csp`. So I guess better solution is just adding "?" to the url. Updated!